src/pyhf/optimize/opt_minuit.py

```diff
@@ -1,20 +1,20 @@
 """Minuit Optimizer Class."""
-from .. import default_backend, exceptions
+from .. import exceptions
 from .mixins import OptimizerMixin
 import scipy
 import iminuit
 
 
 class minuit_optimizer(OptimizerMixin):
     """
-    Optimizer that uses iminuit.Minuit.migrad.
+    Optimizer that minimizes via :func:`iminuit.Minuit.migrad`.
```

Comment:
I'm not sure why, but this isn't getting linked to https://iminuit.readthedocs.io/en/stable/reference.html#iminuit.Minuit.migrad properly

Reply:
Unsure either.

```
$ python -m sphinx.ext.intersphinx <(curl -s https://iminuit.readthedocs.io/en/stable/objects.inv)
```

this shows me

```
py:method
	... 
	iminuit.Minuit.migrad                    reference.html#iminuit.Minuit.migrad
	... 
```

which indicates that `migrad` is there correctly as a function.

Reply:
Strange. :/ Well, maybe we can make this an Issue to look more at later after this is merged?